sdks/java/io/elasticsearch-tests/elasticsearch-tests-5/pom.xml

```diff
@@ -75,6 +75,22 @@
     </dependencies>
     <build>
         <plugins>
+             <plugin>
+			     <groupId>org.apache.maven.plugins</groupId>
+			        <artifactId>maven-dependency-plugin</artifactId>
+			        <executions>
+			          <execution>
+			            <goals>
+			              <goal>analyze-only</goal>
+			            </goals>
+			            <configuration>
+			              <ignoredUsedUndeclaredDependencies>
+			                <ignoredUsedUndeclaredDependency>org.hamcrest:hamcrest-all:jar:${hamcrest.version}</ignoredUsedUndeclaredDependency>
```

Comment:
Why an exception for hamcrest-all is needed here ? is this somehow transitive? or just a typo?
Also if you work on the sub-sequent PR to ban the deps it would be nice to fix this indentation (and the one in the nexmark pom.xml file).

Reply:
@iemejia FYI, I fixed the indentation before merging.